examples/dog-app/README.md

```diff
@@ -0,0 +1,9 @@
+# Dog app
+
+<p class="description">An app that shows dog images based on selected breeds or sub-breeds. </p>
+
+A basic Toolpad app that shows how to call an API and work with data grid and image component.
+
+To build this app step-by-step, visit the [docs](https://mui.com/toolpad/getting-started/quickstart/#building-your-first-application)
+
+[Open in CodeSandbox](https://codesandbox.io/p/sandbox/github/mui/mui-toolpad/tree/master/examples/dog-app)
```

Comment:
Preview of the dog app example: https://3g6k84-3000.csb.app/preview/pages/PCYkw5V

Reply:
This link doesn't work for me:

<img width="618" alt="Screenshot 2023-06-07 at 8 57 38 PM" src="https://github.com/mui/mui-toolpad/assets/92228082/db93ab19-8357-45bf-88cc-6529299c1192">

Reply:
> This link doesn't work for me:
> 
> <img alt="Screenshot 2023-06-07 at 8 57 38 PM" width="618" src="https://user-images.githubusercontent.com/92228082/244118138-db93ab19-8357-45bf-88cc-6529299c1192.png">

Interesting, I guess the codesandbox build link expires soon. This should work instead, but it will build the app when you visit it: https://codesandbox.io/p/sandbox/github/mui/mui-toolpad/tree/landing_page_add_examples/examples/dog-app